### PR TITLE
Draft: Wallaby package refresh

### PR DESCRIPTION
**Pull request by cityofships**

### CODE REVIEW

File: etc/kayobe/dnf.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@
 # dnf_config:
 #   proxy: http://proxy.example.com
 #dnf_config:
+dnf_config:
+  install_weak_deps: False
```
Comment (markgoddard) [inline etc/kayobe/dnf.yml:12]: Why is this required?

Comment (cityofships) [reply etc/kayobe/dnf.yml:12]: It's to keep hosts free of crap like PackageKit

**Comment by markgoddard** [comment]
Stale
